exercises/protein-translation/tests/proteins.rs

```diff
@@ -3,42 +3,41 @@ extern crate protein_translation as proteins;
 #[test]
 fn test_methionine() {
     let info = proteins::parse(make_pairs());
-    assert_eq!(info.name_for("ATG"), Ok("methionine"));
+    assert_eq!(info.name_for("AUG"), Ok("methionine"));
 }
 
 #[test]
 #[ignore]
 fn test_cysteine_tgt() {
     let info = proteins::parse(make_pairs());
-    assert_eq!(info.name_for("TGT"), Ok("cysteine"));
+    assert_eq!(info.name_for("UGU"), Ok("cysteine"));
 }
 
 #[test]
 #[ignore]
 fn test_stop() {
     let info = proteins::parse(make_pairs());
-    assert_eq!(info.name_for("TAA"), Ok("stop codon"));
+    assert_eq!(info.name_for("UAA"), Ok("stop codon"));
 }
 
 #[test]
 #[ignore]
 fn test_valine() {
     let info = proteins::parse(make_pairs());
-    assert_eq!(info.name_for("GTT"), Ok("valine"));
+    assert_eq!(info.name_for("GUU"), Ok("valine"));
 }
 
 #[test]
 #[ignore]
 fn test_isoleucine() {
     let info = proteins::parse(make_pairs());
-    assert_eq!(info.name_for("ATT"), Ok("isoleucine"));
+    assert_eq!(info.name_for("AUU"), Ok("isoleucine"));
 }
 
 #[test]
 #[ignore]
 fn test_arginine_name() {
     let info = proteins::parse(make_pairs());
-    assert_eq!(info.name_for("CGA"), Ok("arginine"));
```

Comment:
removal intentional? I notice `CGA` is still an arginine in https://en.wikipedia.org/wiki/Genetic_code#RNA_codon_table